.github/workflows/verify.yml

```diff
@@ -0,0 +1,77 @@
+name: Verify
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    timeout-minutes: 40
+
+    services:
+      postgres:
+        image: postgres:9.6
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby:
+          - 2.5
+          - 2.6
+          - 2.7
+
+    env:
+      RAILS_ENV: test
+
+    name: Ruby ${{ matrix.ruby }}
+    steps:
+      - name: Install system dependencies
+        run: sudo apt-get install libpcap-dev graphviz
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+
+      - name: Setup bundler
+        run: |
+          gem install bundler
+
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Bundle install
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+
+      - name: Test
+        run: |
+          cp spec/dummy/config/database.yml.github_actions spec/dummy/config/database.yml
+          bundle exec rake --version
+          bundle exec rake db:create db:migrate
+
+          # Disabling this check because it is proving unreliable
+          # git diff --exit-code spec/dummy/db/structure.sql
+          bundle exec rake spec
+          bundle exec rake yard
```

Comment:
Ran this gem's tests locally and it generates the coverage artifacts under the coverage folder:
![image](https://user-images.githubusercontent.com/60357436/116388528-d2d5bd00-a813-11eb-8cba-807019b9b311.png)

I think we should be able to add this step and have it available for download:
```
      - name: Upload coverage report
        uses: actions/upload-artifact@v2
        with:
          name: coverage
          path: |
            coverage/
          retention-days: 1
```

Reply:
Looks like it'll need to be namespaced for each ruby version, maybe:
```
      - name: Upload coverage report
        uses: actions/upload-artifact@v2
        with:
          name: coverage-${{ matrix.ruby }}
          path: |
            coverage/
          retention-days: 1
```

Reply:
Working now, thanks!

![image](https://user-images.githubusercontent.com/60357436/116389974-7f646e80-a815-11eb-8dc6-45f95f6161c2.png)